tools/cli/commands/generate.js

```diff
@@ -415,6 +456,110 @@ function generatePlugin( answers, pluginDir ) {
 	writeToFile( pluginDir + '/README.txt', readmeTxtContent + readmeTxtData );
 }
 
+/**
+ * Generate js-package files
+ *
+ * @param {object} answers - Answers from questions.
+ * @param {string} pkgDir - Github action directory path.
+ */
+function generateJsPackage( answers, pkgDir ) {
+	const ts = answers.typescript.startsWith( 'ts' );
+	let filename, opts, xtends;
+
+	if ( ts ) {
+		filename = 'tsconfig.json';
+		opts = {
+			typeRoots: [ './node_modules/@types/', 'src/*' ],
+			outDir: './build/',
+		};
+		switch ( answers.typescript ) {
+			case 'ts-src':
+				xtends = '\n\t"extends": "jetpack-js-tools/tsconfig.base.json",';
+				break;
+			case 'ts-webpack':
+				xtends = '\n\t"extends": "jetpack-js-tools/tsconfig.tsc-definitions.json",';
```

Comment:
Is this `tsc-definitions` file auto-generated somehow? I can't find it in the repo.

Reply:
No, it's the wrong name.